src/main/java/com/conveyal/datatools/manager/gtfsplus/CalendarAttribute.java

```diff
@@ -0,0 +1,20 @@
+package com.conveyal.datatools.manager.gtfsplus;
+
+import com.conveyal.gtfs.model.Entity;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class CalendarAttribute extends Entity {
+
+    private static final long serialVersionUID = 1L;
+
+    public String service_id;
+    public String service_description;
+
+    // TODO
+    @Override
+    public void setStatementParameters(PreparedStatement statement, boolean setDefaultId) throws SQLException {
```

Comment:
What's this for? Wait is the TODO all about?

Reply:
That was just auto-generated. Will remove and throw a `UnsupportedOperationException`.